test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/ThreadCountTests.cs

```diff
@@ -16,19 +15,12 @@ public class ThreadCountTests
     {
         public async Task ZeroToTenThreads(int threadCount)
         {
-            var config = new ConfigurationBuilder()
-                .AddInMemoryCollection(new Dictionary<string, string>
-                {
-                    { "server.urls", $"http://localhost:0/" }
-                })
-                .Build();
-
             var hostBuilder = new WebHostBuilder()
-                .UseConfiguration(config)
                 .UseKestrel(options =>
                 {
                     options.ThreadCount = threadCount;
                 })
+                .UseUrls("http://localhost:0/")
```

Comment:
Should this be changed to `127.0.0.1:0`?  How is it even passing with `localhost:0`?